lib/src/chart/bar_chart/bar_chart_data.dart

```diff
@@ -768,16 +768,21 @@ class BarTooltipItem with EquatableMixin {
   /// TextStyle of the showing content.
   final TextStyle textStyle;
 
+  /// List<InlineSpan> for more flexibility in showing content.
+  final List<InlineSpan> children;
+
   /// content of the tooltip, is a [text] String with a [textStyle].
-  BarTooltipItem(String text, TextStyle textStyle)
+  BarTooltipItem(String text, TextStyle textStyle, [List<InlineSpan> children])
```

Comment:
Why did you put List<InlineSpan> around [ ] ?

Reply:
i figured this is the easiest way to do it without breaking any compatibility.  could have also done named.  i have no preference

Reply:
so you can still call `BarTooltipItem('something', TextStyle(fontSize: 12))` with no issue

Reply:
Okay please change it to named parameter.
Also, it will be great if you do the same in `LineTooltipItem`, and `ScatterTooltipItem`.